common/src/main/java/org/opensearch/ml/common/transport/deploy/MLDeployModelInput.java

```diff
@@ -26,8 +30,10 @@ public class MLDeployModelInput implements Writeable {
     private MLTask mlTask;
 
     public MLDeployModelInput(StreamInput in) throws IOException {
+        Version streamInputVersion = in.getVersion();
         this.modelId = in.readString();
         this.taskId = in.readString();
+        this.tenantId = streamInputVersion.onOrAfter(VERSION_2_19_0) ? in.readOptionalString() : null;
```

Comment:
(Style) I guess this works but I've nearly always seen the new fields added at the end of the stream. 

(Same comment on the `writeTo()` and `MLForwardInput` later.)

Since they match it won't create a problem, but from a readability standpoint it requires some extra vigilance.

Reply:
Agree. Addressing this.